cmd/vulcan-vulners/main.go

```diff
@@ -327,7 +328,10 @@ func analyzeReport(target string, nmapReport *gonmap.NmapRun) ([]report.Vulnerab
 	return vulns, nil
 }
 
-func run(ctx context.Context, target string, optJSON string, state state.State) (err error) {
+func run(ctx context.Context, target, assetType, optJSON string, state checkstate.State) (err error) {
+	l := check.NewCheckLog(checkName)
+	logger = l.WithFields(logrus.Fields{"target": target, "options": optJSON})
```

Comment:
Do we add the asset type to the logged fields?

Reply:
Yes! I missed this one :+1: